api/config/config_preprod.json

```diff
@@ -133,8 +133,8 @@
     "clientId": ""
   },
   "accessRenewal": {
-    "expiryDays": 365,
-    "sendEmails": false,
+    "expiryDays": 3650,
```

Comment:
perf I think is correct indefinitely @ 10y, preprod should probably eventually be 1y for consistency, but fine with this for now if we're not sure of the impact.